src/plugins/formulas/engine/settings.js

```diff
@@ -5,9 +5,6 @@ export const DEFAULT_LICENSE_KEY = 'internal-use-in-handsontable';
 const DEFAULT_SETTINGS = {
   licenseKey: DEFAULT_LICENSE_KEY,
 
-  binarySearchThreshold: 20,
-  matrixDetection: false,
-  matrixDetectionThreshold: 100,
   useColumnIndex: false,
```

Comment:
All removed options are correct, but we have to add new option as well
```suggestion
  useArrayArithmetic: false,
  useColumnIndex: false,
```

Reply:
https://github.com/handsontable/handsontable/pull/8669/commits/aadcbf01170905b230c8ab35424b574781e7aad1